hooks/command

```diff
@@ -269,8 +269,13 @@ if [[ -n "${BUILDKITE_PLUGIN_DOCKER_SHM_SIZE:-}" ]]; then
 fi
 
 # Handle entrypoint if set (or empty), and default shell to disabled
-if [[ "${BUILDKITE_PLUGIN_DOCKER_ENTRYPOINT-false}" != "false" ]] ; then
-  args+=("--entrypoint" "${BUILDKITE_PLUGIN_DOCKER_ENTRYPOINT:-}")
+if [[ "${BUILDKITE_PLUGIN_DOCKER_ENTRYPOINT-unset}" != "unset" ]] ; then
+  # Reset the entrypoint with an empty string if it's false-y
+  if [[ "${BUILDKITE_PLUGIN_DOCKER_ENTRYPOINT:-false}" =~ ^(false|off|0)$ ]] ; then
+    args+=("--entrypoint" "")
+  else
+    args+=("--entrypoint" "${BUILDKITE_PLUGIN_DOCKER_ENTRYPOINT:-}")
```

Comment:
I think we should only set `shell_disabled=1` here, not on `false`, otherwise we cannot run commands like `sh -c -e ""` that are multiline